_data/sites.yml

```
@@ -1,3 +1,8 @@
+- domain: sothawo.com
+  url: https://www.sothawo.com/
+  size: 85.14
+  last_checked: 2024-05-03
```

```suggestion
  last_checked: 2024-04-03
```

You're from the future! 😄

---

oops, thanks for fixing this!